src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/PhpFrameworkExtensionTest.php

```diff
@@ -272,5 +272,6 @@ public static function emailValidationModeProvider()
         foreach (Email::VALIDATION_MODES as $mode) {
             yield [$mode];
         }
+        yield ['loose'];
```

Comment:
Please revert this. Based on my follow-up commit, `loose` is already part of `Email::VALIDATION_MODES` in the 6.4 branch.